### PR TITLE
Split unreliable network milestone

### PR DESCRIPTION
**Pull request by fryorcraken**

### CODE REVIEW

**Review by Ivansete-status — APPROVED** [review]

**Review by chaitanyaprem — APPROVED** [review]
LGTM

**Review by chair28980 — APPROVED** [review]
LGTM

File: ROADMAP.md [hunk 1]
```diff
@@ -358,20 +358,31 @@ This is part of the general SDK strategy and expected to be implemented in each
...
-- Hackathon with poor internet connectivity
 - prevent incoming connections
 - Usage of proxy or VPN to access Internet
 - Mix of WAN/LAN configuration.
 - Usage of IPV6
 
 Note: Bluetooth, network mesh and internet outage are not in the scope of this milestone. 
 
-### TLS-less Browser Connections
+### FQDN-less Browser Connections
```
Comment (chair28980) [inline ROADMAP.md:385]: @fryorcraken fqdn?

Comment (chair28980) [reply ROADMAP.md:385]: ok, i looked it up, got it

**Pull request merged**